client/src/app/site/assignments/components/assignment-detail/assignment-detail.component.html

```diff
@@ -148,15 +148,20 @@ <h3>{{ 'Candidates' | translate }}</h3>
                     >
                         <!-- implicit item references into the component using ng-template slot -->
                         <ng-template let-item>
-                            <span *ngIf="hasPerms('manage')">
-                                <button
-                                    mat-icon-button
-                                    matTooltip="{{ 'Remove candidate' | translate }}"
-                                    (click)="removeCandidate(item)"
-                                >
-                                    <mat-icon>clear</mat-icon>
-                                </button>
-                            </span>
+                            <div class="single-candidate-line">
+                                <ng-container *ngIf="hasPerms('manage')">
```

Comment:
This was a bug in the original code. Move "hasPerms" to the `remove-candidate` button